eng/Build.props

```diff
@@ -1,6 +1,21 @@
 <Project>
   <Import Project="Common.props" />
 
+  <!--
+    Copied from the Arcade SDK but w/ a nuance for npm projects. If specific projects need to build separately,
+    override their metadata below.
+  -->
+  <ItemDefinitionGroup>
+    <ProjectToBuild>
+      <!-- Can use Extension metadata as a value source but not in Conditions. -->
+      <_Extension>%(Extension)</_Extension>
+      <BuildInParallel Condition=" '%(_Extension)' != '.npmproj' ">false</BuildInParallel>
+      <BuildInParallel Condition=" '%(_Extension)' == '.npmproj' ">true</BuildInParallel>
+      <_Extension />
+      <RestoreInParallel>%(BuildInParallel)</RestoreInParallel>
+    </ProjectToBuild>
+  </ItemDefinitionGroup>
```

Comment:
Confirmed locally. Arcade builds and restores `npm` projects in a separate batch from everything else with `BuildInParallel=false` given to the `<MSBuild />` task.

Reply:
Oops, this is backwards on one hand and `%(Extension)` is always exactly `"%(Extension)"` on the other ☹️ So, every project was getting `%(BuildInParallel` and `%(RestoreInParallel)` set to `false`. It's as if well-known metadata can't be used at all in an `<ItemDefinitionGroup/>`.

IOW my experiments were clean but this change did too much.

@rainersigwald any suggestions❔